src/lib/datepicker/datepicker-content.html

```diff
@@ -6,5 +6,6 @@
     [maxDate]="datepicker._maxDate"
     [dateFilter]="datepicker._dateFilter"
     [selected]="datepicker._selected"
-    (selectedChange)="datepicker._selectAndClose($event)">
+    (selectedChange)="datepicker._selectAndClose($event)"
```

Comment:
This `_selectAndClose` method and its comment should be updated since it no longer closes